src/planscape/forsys/generate_projects_for_a_single_scenario.R

```diff
@@ -62,5 +62,12 @@ generate_projects_for_a_single_scenario <- function(forsys_input_data,
     )
   )
 
+  # Adds the input geo_wkt column to the stand output df.
+  input_stand_ids_and_geometries = select(forsys_input_data,
+                                          c(stand_id_field, geo_wkt_field)) 
+  run_outputs$stand_output <- merge(run_outputs$stand_output, 
+                                    input_stand_ids_and_geometries,
+                                    by=stand_id_field)
+
```

Comment:
note: I also tried merging geometries at this stage instead of in python.

For the sake of documentation, here's what I tried:
```
library(rgeos)

output_project_and_stands_and_geometries = select(run_outputs$stand_output,
  c(proj_id_field, stand_id_field, 'geometry')) %>% 
  group_by('stand_id') %>% 
  summarise (merged_geometry = gUnaryUnion('geometry'))
run_outputs$project_output <- merge(run_outputs$project_output,
  output_project_and_stands_and_geometries,
  by=proj_id_field)
```

This failed because of a limitation in gUnaryUnion:
```
Caused by error in `gUnaryUnion()`:
! Invalid geometry, may only be applied to polygons
```

I looked in https://github.com/OurPlanscape/Planscape/blob/main/analysis/code/forsys_tcsi_testing.md to see whether I was missing something obvious, but didn't see any documentation of polygon merging logic.

Instead of digging into it more, I opted to use the merge_polygons function in #554.

Reply:
I suspect it's because of the `'` around geometry but fair enough.

Reply:
Acknowledged.